packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/Filtering/FilterFlyoutButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import FilterFlyout from './FilterFlyout';
+
+// TODO: Do not use
+const handleNumberInput = ({ event, instance, column }) => {
+  const { data, onDataUpdate, setFilter } = instance;
+  console.log({ column, instance });
```

Comment:
Remove console log

Reply:
This file was no being used anyways so I deleted

Reply:
👍🏻